auth.py

```diff
@@ -23,9 +23,17 @@ def add_user(user, password, pwdb):
         pwdb[user] = password
     return pwdb
 
+def authenticate (user, password, pwdb):
+    if user in pwdb.keys():
```

Comment:
```suggestion
    if user in pwdb:
```
you don't need to use `.keys()`, this should also work